src/shared/utils/app/comment-to-post-sort-type.ts

```diff
@@ -0,0 +1,42 @@
+import { CommentSortType, PostSortType } from "lemmy-js-client";
+
+function assertType<T>(_: T) {}
+
+export default function commentToPostSortType(
+  sort: PostSortType | CommentSortType,
```

Comment:
Why does this need to handle either type? You don't need to convert from a PostSortType to a PostSortType.

Reply:
I initially had `sort` for Home and Community as the mixed sort type.